live/live-root/root/.mozilla/firefox/profile/user.js.template

```diff
@@ -20,6 +20,14 @@ user_pref("browser.startup.homepage_override.mstone", "ignore");
 // TODO: make the XDG portal working again, it has more features (mounting USB flash)
 user_pref("widget.use-xdg-desktop-portal.file-picker", 0);
 
+// configure devtools
+// show devtools in a separate window
+user_pref("devtools.toolbox.host", "window");
+// show the console tab instead of the page inspector after pressing F12
+user_pref("devtools.toolbox.selectedTool", "console");
+// show time stamps in the console
+user_pref("devtools.webconsole.timestampMessages", true);
```

Comment:
For saving more space, I guess you could even show only errors by disabling others

```
devtools.webconsole.filter.info	        false	
devtools.webconsole.filter.warn       false   
```

Reply:
We use logging at very few places in the web UI, most of them are errors or warnings already. We can change the filtering later when really needed., for now I'd keep it as it is.